test/UID2.Client.Test.Utils/UID2TokenGenerator.cs

```diff
@@ -78,7 +82,22 @@ public static string GenerateUID2TokenV2(string uid, Key masterKey, int siteId,
 
         public static string GenerateUID2TokenV3(string uid, Key masterKey, int siteId, Key siteKey)
         {
-            return GenerateUID2TokenV3(uid, masterKey, siteId, siteKey, DefaultParams);
+            return GenerateUID2TokenWithDebugInfo(uid, masterKey, siteId, siteKey, DefaultParams, false);
+        }
+        
+        public static string GenerateUID2TokenV3(string uid, Key masterKey, int siteId, Key siteKey, Params encryptParams)
+        {
+            return GenerateUID2TokenWithDebugInfo(uid, masterKey, siteId, siteKey, encryptParams, false);
+        }
+        
+        public static string GenerateUID2TokenV4(string uid, Key masterKey, int siteId, Key siteKey)
```

Comment:
we can remove (or make private) these versions that don't take Params, we're always using Params